kraken-app/src/Components/CardViewPage/CardViewPage.jsx

```diff
@@ -25,7 +26,7 @@ function CardViewPage(props) {
             </Grid>
             <Grid item xs={5}>
                 <Typography variant="h4" component="div" gutterBottom>
-                    
```

Comment:
We only have one name right now so this section can be removed

Reply:
Shouldn't it say the name then?

Reply:
What is the one name?